addons/base_automation/models/base_automation.py

```diff
@@ -53,20 +53,20 @@ class BaseAutomation(models.Model):
                                     You can put a negative number if you need a delay before the
                                     trigger date, like sending a reminder 15 minutes before a meeting.""")
     trg_date_range_type = fields.Selection([('minutes', 'Minutes'), ('hour', 'Hours'), ('day', 'Days'), ('month', 'Months')],
-                                           string='Delay type', default='day')
+                                           string='Delay type', default='hour')
     trg_date_calendar_id = fields.Many2one("resource.calendar", string='Use Calendar',
                                             help="When calculating a day-based timed condition, it is possible to use a calendar to compute the date based on working days.")
     filter_pre_domain = fields.Char(string='Before Update Domain',
                                     help="If present, this condition must be satisfied before the update of the record.")
     filter_domain = fields.Char(string='Apply on', help="If present, this condition must be satisfied before executing the action rule.")
     last_run = fields.Datetime(readonly=True, copy=False)
-    on_change_fields = fields.Char(string="On Change Fields Trigger", help="Comma-separated list of field names that triggers the onchange.")
-    trigger_field_ids = fields.Many2many('ir.model.fields', string='Watched fields',
+    on_change_field_ids = fields.Many2many('ir.model.fields', 'model_id', string="On Change Fields Trigger", help="Fields that trigger the onchange.")
```

Comment:
Oh boy!
The second argument of `fields.Many2many` is the name of the relation table. We end up with a table named `model_id`.

@odony do you mind if fix this even in saas~13.1?
@rco-odoo To avoid other stupid typos like this, what about forcing these arguments to be keyword-only?

Reply:
yes, let's fix it in 13.1

Reply:
Done with #45283